common/app/views/fragments/header.scala.html

```diff
@@ -154,7 +154,7 @@
                 </div>
             </div>
 
-            @fragments.nav.navigation(page)
+            @fragments.nav.mainNavigation(page)
```

Comment:
Had to change the name of this fragment so I could easily import the navigation package in templates (instead of importing this template)